Resources/Prototypes/Catalog/Fills/Boxes/emergency.yml

```diff
@@ -216,6 +228,10 @@
     - id: Flare
     - id: FoodBreadBaguette
     - id: DrinkWaterBottleFull
+  - type: Sprite
+    layers:
+    - state: internals
+    - state: nitrogentank
   - type: Label
     currentLabel: reagent-name-nitrogen
 
```

Comment:
Also fix BoxSurvivalSyndicateNitrogen in line 289

Reply:
Sure thing.